diff --git a/modules/mod-mythic-plus/data/sql/db-world/b_mythic_plus_capable_dungeon.sql b/modules/mod-mythic-plus/data/sql/db-world/b_mythic_plus_capable_dungeon.sql
index 64399af..e10d9db 100644
--- a/modules/mod-mythic-plus/data/sql/db-world/b_mythic_plus_capable_dungeon.sql
+++ b/modules/mod-mythic-plus/data/sql/db-world/b_mythic_plus_capable_dungeon.sql
@@ -7,16 +7,14 @@ CREATE TABLE `mythic_plus_capable_dungeon`(
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
 
-INSERT INTO `mythic_plus_capable_dungeon` (`map`, `mapdifficulty`, `final_boss_entry`) VALUES (658, 0, 36658);
-INSERT INTO `mythic_plus_capable_dungeon` (`map`, `mapdifficulty`, `final_boss_entry`) VALUES (632, 0, 36502);
-INSERT INTO `mythic_plus_capable_dungeon` (`map`, `mapdifficulty`, `final_boss_entry`) VALUES (619, 0, 29311);
-INSERT INTO `mythic_plus_capable_dungeon` (`map`, `mapdifficulty`, `final_boss_entry`) VALUES (601, 0, 29120);
-INSERT INTO `mythic_plus_capable_dungeon` (`map`, `mapdifficulty`, `final_boss_entry`) VALUES (600, 0, 26632);
-INSERT INTO `mythic_plus_capable_dungeon` (`map`, `mapdifficulty`, `final_boss_entry`) VALUES (604, 0, 29306);
-INSERT INTO `mythic_plus_capable_dungeon` (`map`, `mapdifficulty`, `final_boss_entry`) VALUES (602, 0, 28923);
-INSERT INTO `mythic_plus_capable_dungeon` (`map`, `mapdifficulty`, `final_boss_entry`) VALUES (599, 0, 27978);
-INSERT INTO `mythic_plus_capable_dungeon` (`map`, `mapdifficulty`, `final_boss_entry`) VALUES (576, 0, 26723);
-INSERT INTO `mythic_plus_capable_dungeon` (`map`, `mapdifficulty`, `final_boss_entry`) VALUES (578, 0, 27656);
-INSERT INTO `mythic_plus_capable_dungeon` (`map`, `mapdifficulty`, `final_boss_entry`) VALUES (574, 0, 23954);
-INSERT INTO `mythic_plus_capable_dungeon` (`map`, `mapdifficulty`, `final_boss_entry`) VALUES (575, 0, 26861);
-INSERT INTO `mythic_plus_capable_dungeon` (`map`, `mapdifficulty`, `final_boss_entry`) VALUES (389, 0, 11519);
-INSERT INTO `mythic_plus_capable_dungeon` (`map`, `mapdifficulty`, `final_boss_entry`) VALUES (47, 0, 4421);
+INSERT INTO `mythic_plus_capable_dungeon` (`map`, `mapdifficulty`, `final_boss_entry`) VALUES (658, 1, 36658);
+INSERT INTO `mythic_plus_capable_dungeon` (`map`, `mapdifficulty`, `final_boss_entry`) VALUES (632, 1, 36502);
+INSERT INTO `mythic_plus_capable_dungeon` (`map`, `mapdifficulty`, `final_boss_entry`) VALUES (619, 1, 29311);
+INSERT INTO `mythic_plus_capable_dungeon` (`map`, `mapdifficulty`, `final_boss_entry`) VALUES (601, 1, 29120);
+INSERT INTO `mythic_plus_capable_dungeon` (`map`, `mapdifficulty`, `final_boss_entry`) VALUES (600, 1, 26632);
+INSERT INTO `mythic_plus_capable_dungeon` (`map`, `mapdifficulty`, `final_boss_entry`) VALUES (604, 1, 29306);
+INSERT INTO `mythic_plus_capable_dungeon` (`map`, `mapdifficulty`, `final_boss_entry`) VALUES (602, 1, 28923);
+INSERT INTO `mythic_plus_capable_dungeon` (`map`, `mapdifficulty`, `final_boss_entry`) VALUES (599, 1, 27978);
+INSERT INTO `mythic_plus_capable_dungeon` (`map`, `mapdifficulty`, `final_boss_entry`) VALUES (576, 1, 26723);
+INSERT INTO `mythic_plus_capable_dungeon` (`map`, `mapdifficulty`, `final_boss_entry`) VALUES (578, 1, 27656);
+INSERT INTO `mythic_plus_capable_dungeon` (`map`, `mapdifficulty`, `final_boss_entry`) VALUES (574, 1, 23954);
+INSERT INTO `mythic_plus_capable_dungeon` (`map`, `mapdifficulty`, `final_boss_entry`) VALUES (575, 1, 26861);
